sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -3457,7 +3457,7 @@ object functions {
    * }}}
    *
    * @param column: the input array column
-   * @param f: col => predicate, the boolean predicate to filter the input column
+   * @param f: col => predicate, the Boolean predicate to filter the input column
```

Comment:
This is fine, though 'boolean' is very commonly used as the noun, despite it being from a name